.azure-pipelines/pipelines.yml

```diff
@@ -86,7 +89,7 @@ stages:
       - env
       checkStageDeps:
       - env
-      checksConcurrency: 10
+      concurrencyChecks: 10
```

Comment:
does these changes make sense? I think the original name is reasonable?

Reply:
i wanted to make all the concurrency vars prefix the same - so its clear what its setting - this makes it more consistent with other vars

ultimately there is no right way - but eg for `authXXX` i really wanted it to be clear that this is an auth setting rather than `myComplicatedSubVarAuth` which is not as immediately obvious and not grouped